packages/taquito-beacon-wallet/test/taquito-beacon-wallet.spec.ts

```diff
@@ -64,4 +70,37 @@ describe('Beacon Wallet tests', () => {
     const formattedParam = await wallet.formatParameters({ gasLimit: 40 });
     expect(formattedParam.gasLimit).toEqual('40');
   });
+
+  it(`Verify removeDefaultParameters for fees`, async () => {
+    const wallet = new BeaconWallet({ name: 'Test', storage: new LocalStorage() });
+    const formattedParam = await wallet.removeDefaultParams({ fee: 10 }, { fee: 30 });
+    expect(formattedParam.fee).toEqual(30);
+  });
+
+  it(`Verify removeDefaultParameters for storageLimit`, async () => {
+    const wallet = new BeaconWallet({ name: 'Test', storage: new LocalStorage() });
+    const formattedParam = await wallet.removeDefaultParams(
+      { storageLimit: 2000 },
+      { storageLimit: 165 }
+    );
+    expect(formattedParam.storageLimit).toEqual(165);
+  });
+
+  it(`Verify removeDefaultParameters for gas limit`, async () => {
+    const wallet = new BeaconWallet({ name: 'Test', storage: new LocalStorage() });
+    const formattedParam = await wallet.removeDefaultParams({ gasLimit: 40 }, { gasLimit: 80 });
+    expect(formattedParam.gasLimit).toEqual(80);
+  });
+
+  it(`Verify disconnect`, async () => {
+    const wallet = new BeaconWallet({ name: 'Test', storage: new LocalStorage() });
+    const disconnected = await wallet.disconnect();
+    expect(disconnected).toBeTruthy;
+  });
+
+  it(`Verify clearActiveAccount`, async () => {
+    const wallet = new BeaconWallet({ name: 'Test', storage: new LocalStorage() });
+    const cleared = await wallet.clearActiveAccount();
+    expect(cleared).toBeTruthy;
```

Comment:
toBeTruthy is missing parentheses `toBeTruthy()`

Reply:
added the ()s.